docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -118,25 +146,137 @@ To retrieve the metadata and playback information on the client side:
    dlog_print(DLOG_DEBUG, LOG_TAG, "Playback State: %d\n", playback_state);
    ```
 
-   The `mc_client_get_playback_state()` function retrieves the playback state from the playback information returned by the `mc_client_get_server_playback_info()` function.
+   The `mc_client_get_playback_state()` retrieves the playback state from the playback information returned by the `mc_client_get_server_playback_info()`.
+
+4. Destroy the media controller client handle using the `mc_client_destroy()`, when media controller client handle is no longer needed:
+
+   ```
+   mc_client_destroy(g_client_h);
+   ```
+
+<a name="get_playlist"></a>
```

Comment:
Global (for consistency): Please avoid using HTML tags in the MD files. Kindly follow the specified hyperlink code for MD.

Reply:
Hello, I have changed hyperlink to the head link for MD. But it does not work on the github.
## Updating and Retrieving Information {: #get-media }
How to make the head link on the github?

Reply:
Hi @jiyong2,

In MD you may use the same name of heading in small case with a '#'. For example:
While linking "Updating and Retrieving Information" to another heading "### Updating Information" somewhere in the document. Use the heading name in small case and instead of space use hyphens "-". As shown in the following example:

```
[Updating and Retrieving Information](#updating-information)
```
Keep the heading name same as it is:

```
### Updating Information
```

Reply:
Okay I will try it.

Reply:
Thanks. The HTML tag changed to head anchor for MD.